backends/bmv2/bmv2stf.py

```diff
@@ -668,8 +735,8 @@ def openInterface(ifname):
                 raise e
```

Comment:
This code (above, not edited) is suspicious:
- sleeping for a fixed time as a normal procedure (e.g. not timeout) seems like bad idea since it increases chances of spurious failures. Would one of the executables finish normally after it runs the test? Can we demote the the `terminate` only to timeout case and try to wait with a timeout (there is `.wait(timeout=...)`)? 
- `wait` after `termninate` might still block indefinetly if the executable ignores `SIGTERM`. In case of timeout, there should probably be attempt to `SIGTERM` it, than wait for some time and then `SIGKILL` it (`.kill()`)
- this would better be done with terminate/kill in `finally` probably